nodeadm/internal/system/_assets/eks-80-ec2.network.template

```diff
@@ -0,0 +1,24 @@
+[Match]
+Driver=ena ixgbevf vif
```

Comment:
Do we still need `Driver=...` ? Just MacAddress should be enough right? At least with "Name", I was able to remove this..

Reply:
we don't need it, but want to keep consistent with the default ec2's one as much as possible.